src/tbb/scheduler_common.h

```diff
@@ -354,6 +354,16 @@ struct suspend_point_type {
     //! Associated coroutine
     co_context m_co_context;
 
+    //! The flag is raised when suspend functor should be called on same thread
+    bool my_is_suspend_transactional{false};
+    //! The flag required to protect case when my_is_suspend_transactional == true for synchronization of suspend finish and resume call
+    std::atomic<bool> my_suspend_protector{false};
+
+    void clear() {
```

Comment:
Why does clear unset just two fields of suspend point class? Consider pick up meaningful name.